### ISSUE TITLE
508-defect-1 [KEYBOARD, SCREENREADER]: After selecting an, focus is not moved to the new content

### ISSUE THREAD

**Issue by noahgelman** [department-of-veterans-affairs/va.gov-team#18716]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Noah Gelman

## User Story or Problem Statement

When the user selects a question to ask the chatbot with keyboard, keyboard focus is not moved to the chatbot response. For users who are blind, and use keyboard to navigate, it means that they select a question, the screen reader reads all the new content, and then they have to navigate through the rest of the previous content to get to the new content, and it's very confusing.

When selecting an option, focus needs to be moved to the new content so the user doesn't have to continue to navigate through old content to get to the new content

## Acceptance Criteria

- [ ] When an option is selected, move keyboard focus to the new chatbot response, so the user can navigate through the new content

**Comment by shefalinayak** [comment]
@noahgelman Do you have any guidance for how to fix this issue? It looks like after a button is selected, and the new content is read out, using arrow keys moves screenreader focus to the next (now disabled) button. Should the disabled buttons be skipped by the screenreader?

**Comment by rumyeahs** [comment]
@shefalinayak Yes that is correct. Once the selection is made, the screenreader is moving on to the response. However it appears the keyboard focus is still on the previous (now disabled) selection and hence when the user navigates next, it re-reads the previous section. Ideally the keyboard focus should shift to the response, alongwith the screenreader focus, when the user has made a selection.
@noahgelman please add if something is missing.

**Comment by shefalinayak** [comment]
@noahgelman We've fixed the issue, and it's ready for review in staging.
